### ISSUE TITLE
Factory contract owned by Gov Contract ?

### ISSUE THREAD

**Issue by narawit0** [Mirror-Protocol/mirror-contracts#21]
The Factory contract  owned by Gov Contract, so the owner address of Factory contract is Gov Contract Address.

How can i sign a signature when i want to execute msg on The Factory Contract to whitelist new mAsset?

**Comment by YunSuk-Yeo** [comment]
Yes owned by Gov Contract, so need to register poll to Gov Contract.
When the poll passed, the execute_msg is executed to register whitelist new asset
